GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -503,6 +503,18 @@ - (void)testRestorePreviousSignInWhenSignedOut {
   [_authState verify];
 }
 
+- (void)testNotRestorePreviousSignWhenCompletionIsNil {
+  [[[_authorization expect] andReturn:_authState] authState];
+  [[[_authState expect] andReturnValue:[NSNumber numberWithBool:NO]] isAuthorized];
+  _completionCalled = NO;
+  _authError = nil;
+
+  [_signIn restorePreviousSignInWithCompletion:nil];
+  [NSThread sleepForTimeInterval:1.0];
+
+  XCTAssertNil(_signIn.currentUser);
+}
```

Comment:
It appears that this test case mimics the previous `testRestorePreviousSignInWhenSignedOut` case except the callback is nil. The previous case tests that the user is "signed out", meaning that they don't have a stored token, therefore their sign in cannot be restored. IMO, it's more interesting to test the scenario that the user does have a stored token, and `restorePreviousSignInWithCompletion` succeeds with a nil callback. However, there isn't an existing standalone test case that tests the success scenario with a non-nil callback, so I think it's ok if you don't add a unit test in this PR.